operate/Makefile

```diff
@@ -3,35 +3,35 @@
 .PHONY: env-up
 env-up:
 	docker-compose -f docker-compose.yml up -d elasticsearch zeebe \
-	&& mvn install -DskipTests=true -Dskip.fe.build=false \
+	&& mvn install -DskipTests=true -Dskip.fe.build=false -DskipChecks \
 	&& CAMUNDA_OPERATE_TASKLIST_URL=http://localhost:8081 \
-	   mvn -f webapp/pom.xml exec:java -Dexec.mainClass="io.camunda.operate.Application" -Dspring.profiles.active=dev,dev-data,auth
+	   mvn clean install -DskipTests -DskipChecks -f ../dist/pom.xml exec:java -Dexec.mainClass="io.camunda.operate.StandaloneOperate" -Dspring.profiles.active=dev,dev-data,auth
```

Comment:
❓ Did you tried it with the `../` does it work?

Reply:
On Windows, it works 😄  Do you expect any issues with that? Doesn't it work for you?

Reply:
The only thing to consider is that initially, the engineers must build the entire project locally (when executing this command the first time after merging). But this must happen only once. (Depending on the upcoming changes, this might be necessary again)

Reply:
I did a test locally, and it still works for me.